vector/src/main/AndroidManifest.xml

```diff
@@ -144,8 +144,8 @@
 
                 <data android:scheme="https" />
                 <data android:host="riot.im" />
-                <data android:host="element.io" />
-                <data android:pathPrefix="/config/" />
+                <data android:host="app.element.io" />
```

Comment:
We should also handle `develop.element.io` and `staging.element.io` links. Please add theme here and in SUPPORTED_HOSTS